### ISSUE TITLE
Xamarin Android support/documentation

### ISSUE THREAD

**Issue by kkarakk** [microsoft/MapsSDK-Native#7]
How do I Use bing maps sdk on xamarin android? There is no tutorial/documentation related to this? Do i just need to port the library/do the Xamarin bindings myself?

**Comment by JuanPapas** [comment]
Xamarin is currently not supported, but we are considering it as a feature request.
We have been getting similar feedback from various folks and we are definitely interested on supporting higher level abstractions like Xamarin for our native controls.

**Comment by Dipak26** [comment]
How Do I use bing maps sdk in React-Native (Android and iOS) ? Do I need to create and wrapper around those sdk to create and React-native module for the bing maps?

**Comment by JuanPapas** [comment]
> How Do I use bing maps sdk in React-Native (Android and iOS) ? Do I need to create and wrapper around those sdk to create and React-native module for the bing maps?
Correct.  At the moment, we don't have wrappers around the native controls but it's another interesting item to consider as feature request.  @Dipak26 please feel free to open a separate issue.
